p2p/pubsub/pubsub.go

```diff
@@ -62,6 +62,9 @@ const (
 	// HareProtocol is the protocol id for hare messages.
 	HareProtocol = "/sm/hr/1"
 
+	// BlockCertify is the protocol id for block certification.
+	BlockCertify = "/sm/bc/1"
```

Comment:
i should have paid more attention to the original change :)

we really should remove all useless stuff from this names, they will add bloat on the wire. for example `/sm/` and third slash is not needed. so as long as they are unique we are good

Reply:
shortened